_posts/2018-07-31-craft-boilerplate-for-heroku-and-local.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Craft 3 Heroku Boilerplate'
+author: dimitri-steinel
+layout: post
+mainImage: ""
+mainImageAlt: ""
+--- 
+
+### First contentful paint
+To make sure we can seamlessly work in teams, we have a specific requirements for our hosting environment.
+Currently, our hosting solution of choice is [Netlify](https://www.netlify.com/) and [Heroku](https://www.heroku.com/). 
+Netlify is mostly for our static websites. Heroku is for almost all other projects.
+With [Craft 3](https://craftcms.com/news/craft-3), which was released in April’18, it is already much easier to install the CMS as [composer package](https://packagist.org/packages/craftcms/cms) and host it. But there were still some things we need to take care of in order to host it on Heroku.
```

Comment:
With [Craft 3](https://craftcms.com/news/craft-3), which was released in April’18, __it's__ already much easier to install the CMS as __a__ [composer package](https://packagist.org/packages/craftcms/cms) and host it. But there were still some things we __needed__ to take care of in order to host it on Heroku.